cla_backend/apps/cla_butler/management/commands/diversity_data_reencrypt.py

```diff
@@ -0,0 +1,57 @@
+import os
+from django.core.management.base import BaseCommand
+from cla_butler.tasks import DiversityDataReencryptTask
+from cla_butler.models import DiversityDataCheck, ACTION
+from legalaid.utils import diversity
+
+
+class Command(BaseCommand):
+    help = "Re-encrypt diversity data"
+    # Number of records processed by each celery task
+    chunk_size = 1000
```

Comment:
I found it confusing that this command has a self.chunk_size and the diversity data check uses chunk_size as a variable but passes in 1000

Reply:
I am not sure I follow this, can you link to other code that you are referencing?

Reply:
Walked through this and agreed to keep as is